### DISCUSSION TITLE
Flash model (material) by mouse picking

### DISCUSSION THREAD

**Discussion by phovorka** [google/model-viewer#3243] [Q&A]
Im trying to find a solution how to flash (highlight) part in glb model when click on it. Please help with solution. In Model-Viewer editor it is working by clicking from list box. I dont wanna change the color -just flash.

**Comment by elalish** [comment]
This example gives the basic idea: https://modelviewer.dev/examples/scenegraph/#pickMaterialExample

You may want to add a requestAnimationFrame callback to fade the color over time. You can also find the relevant editor code in our `space-opera` package in the `materials_panel`.

**Reply by pixlbndr** [reply]
Hi @elalish I too would like to implement this behavior where the part of the model flashes when clicked like it does in the editor. I can not find the relevant code as you pointed out above. Can you paste the code here or provide updated instructions on where to find the code? Much appreciated!

**Reply by elalish** [reply]
https://github.com/google/model-viewer/blob/master/packages/space-opera/src/components/materials_panel/materials_panel.ts#L253
